contracts/cryptoblades.sol

```diff
@@ -68,6 +68,9 @@ contract CryptoBlades is Initializable, AccessControlUpgradeable {
 
         // migrateTo_5e833b0
         durabilityCostFight = 1;
+		
+        // migrateTo_???????
+        rerollFightCost = 20;
```

Comment:
This needs to go in a new `migrateTo_[X]`-style function. Name said new function `migrateTo_PLACEHOLDER` for now.